src/Chart.php

```diff
@@ -10,21 +10,24 @@
 use Atk4\Ui\View;
 
 /**
- * ChartJS 2.7.x documentation https://www.chartjs.org/docs/2.7.3/
- * ChartJS 3.9.1 documentation https://www.chartjs.org/docs/3.9.1/.
+ * ChartJS 3.9.1 documentation https://www.chartjs.org/docs/3.9.1/
+ * Chart examples https://www.chartjs.org/docs/latest/samples/information.html.
  */
 class Chart extends View
 {
     /** @var string HTML element type */
     public $element = 'canvas';
 
-    /** @var string Type of chart - bar|pie etc. */
+    /** @var string Type of chart - bar|pie etc. See ChartType class */
     public $type;
 
     /** @var bool should we add JS include into application body? Set "false" if you do it manually. */
     public $jsInclude = true;
 
-    /** @var array<int, array{string, string}> We will use these colors in charts */
+    /** @var string CDN URL */
```

Comment:
```suggestion
    /** @var string */
```

"copying" property name in phpdocs is not helpfull